### DISCUSSION TITLE
（The fuzz testing section within the DeFi project.）There is a parameter order issue here that causes the test to pass

### DISCUSSION THREAD

**Discussion by chengfei867** [Cyfrin/foundry-full-course-cu#1167] [Q&A]
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/64878126/cd074ad0-a00f-4d35-aadb-427b3a0bd955)
The parameters here are the user address first and the token address last.

![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/64878126/007c6d75-9830-4420-9108-d4dc432c0c51)
But when calling here, the order is reversed.

I followed the reverse order in the video and the test results all passed. However, if I use the correct parameter order to test, some tests will fail and the error will be as follows:
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/64878126/d3088936-7c95-4feb-97a5-5da9691cb6b6)

**Comment by 0xusmanf** [comment]
Can you please provide the video time stamp of the total where you faced this problem? And it would be more helpful for us to answer if you can provide your GitHub repo too.

**Reply by chengfei867** [reply]
Okay, here is the timestamp for this question: https://youtu.be/wUjYK5gwNZs?t=14678
And there is my repo:https://github.com/chengfei867/foundry-defi-stablecoin-f23.git
The problem lies in the function redeemCollateral in test/fuzz/Handler.t.sol.
